cmd/options.go

```diff
@@ -381,9 +389,12 @@ func (o *Options) LogOptions() {
 	} else {
 		log.Info().Msg("✨ Running with:")
 	}
+	if o.DryRun {
+		log.Info().Msgf("✨ - dry-run: %t", o.DryRun)
+	}
```

Comment:
I think it's better if you just hide all the comments related to creating or connecting to a cluster. What do you think? 
```go
	if o.DryRun {
		log.Info().Msgf("✨ - dry-run: %t", o.DryRun)
	} else {
		if !o.CreateCluster {
			log.Info().Msgf("✨ - reusing existing cluster")
		} else {
			log.Info().Msgf("✨ - local-cluster-tool: %s", o.clusterProvider.GetName())
			log.Info().Msgf("✨ - cluster-name: %s", o.ClusterName)
			if o.clusterProvider.GetName() == "kind" {
				if o.KindOptions != "" {
					log.Info().Msgf("✨ - kind-options: %s", o.KindOptions)
				}
				if o.KindInternal {
					log.Info().Msgf("✨ - kind-internal: %t", o.KindInternal)
				}
			}
			if o.clusterProvider.GetName() == "k3d" && o.K3dOptions != "" {
				log.Info().Msgf("✨ - k3d-options: %s", o.K3dOptions)
			}
		}
	}
```